cms/templates/emails/utilities_bulk_update_done_message.txt

```diff
@@ -0,0 +1,14 @@
+<%! from django.utils.translation import ugettext as _ %>
+${_("Hello, {user}").format(user=user_username)}
+
+% if success:
+${_("Your bulk update on the {course} course has successfully completed. Previously published problems have been automatically republished.").format(course=course_name)}
```

Comment:
Can we just make this `course_name=course_name`?

Reply:
done